### PR TITLE
Added deepcell application wrapper for model

### PR DESCRIPTION
**Pull request by JLrumberger**
**What is the purpose of this PR?**

Wraps the model with the deepcell `Application` class that does batching and tile & stitch during inference.

**How did you implement your changes**

Added the required functions `cell_preprocess`, `cell_postprocess` and `format_output` as well as class `CellClassification` that inherits from `deepcell.Application`.

**Remaining issues**

None

### CODE REVIEW

File: cell_classification/application.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    normalize = kwargs.get('normalize', True)
+    marker = kwargs.get('marker')
+    normalization_dict = kwargs.get('normalization_dict')
+    if normalize:
+        if marker in normalization_dict.keys():
+            norm_factor = normalization_dict[marker]
+        else:
+            print("Norm_factor not found for marker {}".format(marker))
+            norm_factor = np.quantile(output[..., 0], 0.999)
+        output[..., 0] /= norm_factor
```
Comment (ngreenwald) [inline cell_classification/application.py:27]: Any reason we need to index into the channels dimension?

File: cell_classification/application_test.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+from application import CellClassification
+from model_builder import ModelBuilder
+from segmentation_data_prep_test import prep_object_and_inputs
+import numpy as np
+import tempfile
+import toml
+import os
+
+
+def predict_test():
```
Comment (ngreenwald) [inline cell_classification/application_test.py:10]: We should have a separate test for the normalize function

File: cell_classification/application.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        raise ValueError("Image data must be 4D, got image of shape {}".format(image.shape))
+
+    normalize = kwargs.get('normalize', True)
+    marker = kwargs.get('marker')
+    normalization_dict = kwargs.get('normalization_dict')
+    if normalize:
+        if marker in normalization_dict.keys():
+            norm_factor = normalization_dict[marker]
+        else:
+            print("Norm_factor not found for marker {}".format(marker))
```
Comment (ngreenwald) [inline cell_classification/application.py]:
```suggestion
            print("Norm_factor not found for marker {}, calculating directly from the image. ".format(marker))
```

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good, just a few small ones

File: cell_classification/application.py (hunk 1 above)
Comment (JLrumberger) [reply cell_classification/application.py:27]: We don't want to normalize the binary mask in channel 2.

File: cell_classification/application_test.py (hunk 2 above)
Comment (JLrumberger) [reply cell_classification/application_test.py:10]: Agree!

**Review by ngreenwald — APPROVED** [review]
Looks good!

**Pull request merged**